app/src/main/contrib/components/ChooseAction/ChooseActionPage.tsx

```diff
@@ -12,10 +12,6 @@ export interface ChooseActionPageLocationProps {
 
 export class ChooseActionPageComponent extends React.Component<PageProperties<ChooseActionPageLocationProps>> {
 
-    componentDidMount() {
```

Comment:
Changing the test for this from `expect(mock.called)` to `expect(mock.calls.length).toBe(1)` revealed that this was being called twice! Once in the `PageActionCreators`, and once superfluously here. There are no other changes to app code here, only to tests.